src/components/hooks/use-livechat.tsx

```diff
@@ -1,7 +1,11 @@
 import { useState, useEffect } from 'react'
 import { getClientInformation, getDomain, getUTMData, isBrowser } from 'common/utility'
 
-export const useLivechat = () => {
+type TLC_API = {
+    open_chat_window?: () => void
+}
```

Comment:
@niloo-fs You don’t need to do this, This is already defined in `global.d.ts`.
Please remove this and instead change `LC_API` to `window.LC_API` at line `135` and then remove line `10` as well.